classes/plugins/DOIPubIdExportPlugin.inc.php

```diff
@@ -127,18 +127,18 @@ function getAdditionalFieldNames($hookName, $args) {
 	 */
 	function getUnregisteredArticles($context) {
 		// Retrieve all published submissions that have not yet been registered.
-		$publishedSubmissionDao = DAORegistry::getDAO('PublishedSubmissionDAO'); /* @var $publishedSubmissionDao PublishedSubmissionDAO */
-		$articles = $publishedSubmissionDao->getExportable(
-			$context->getId(),
-			$this->getPubIdType(),
-			null,
-			null,
-			null,
-			$this->getPluginSettingsPrefix(). '::' . DOI_EXPORT_REGISTERED_DOI,
-			null,
-			null
-		);
-		return $articles->toArray();
+		// $publishedSubmissionDao = DAORegistry::getDAO('PublishedSubmissionDAO'); /* @var $publishedSubmissionDao PublishedSubmissionDAO */
```

Comment:
To be completed as part of this issue: https://github.com/pkp/pkp-lib/issues/4867